### DISCUSSION TITLE
Any changes done in UI  version 1.23.2 , Not reflected on my UI

### DISCUSSION THREAD

**Discussion by Shivam22012024** [blockscout/blockscout#9367] [General]
I need a change on my UI, Like change in Footer file, Navbar and tittle. I do changes in my file and then clean my docker and again built it using docker-compose up --build.

But after doing this, No changes reflect on my UI. Why ?

**Comment by GimluCom** [comment]
You can customize your UI/frontend by editing the ENVs: https://github.com/blockscout/frontend/blob/main/docs/ENVS.md

If you want to make changes to the frontend source code, please read this: https://github.com/blockscout/frontend/blob/main/docs/CUSTOM_BUILD.md
